src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+  }
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance

Reply:
you mean not using duration, and in the second instead of transition: transform 0.3 s, color 0.3 s; write a simple transition: 0.3s; ?